catalog/ui/catalog-ui-search/src/main/webapp/component/sort-item/sort-item.less

```diff
@@ -8,9 +8,15 @@
     margin-top: 10px;
   }
 
-  @{customElementNamespace}sort-item:first-of-type {
+  @{customElementNamespace}sort-item:not(:first-of-type) {
     .sort-remove {
-      display: none;
+      display: inline-block;
+    }
+  }
+
+  @{customElementNamespace}sort-item {
+    .sort-direction {
+      width: ~'calc(50% - @{minimumButtonSize} - 40px)';
```

Comment:
Is there a reason we are doing these drop down boxes as side by side even though we do not do that with any other form controls that I am aware of?